src/uct/ib/rdmacm/rdmacm_component.c

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2017-2021.  ALL RIGHTS RESERVED.
+ * Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+ * See file LICENSE for terms.
+ */
+
+#ifdef HAVE_CONFIG_H
+#  include "config.h"
+#endif
+
+#include "rdmacm_cm.h"
+
+static ucs_config_field_t uct_rdmacm_cm_config_table[] = {
+    {"", "", NULL,
+     ucs_offsetof(uct_rdmacm_cm_config_t, super), UCS_CONFIG_TYPE_TABLE(uct_cm_config_table)},
+
+    {"SOURCE_ADDRESS", "",
+     "If non-empty, specify the local source address (IPv4 or IPv6) to use \n"
+     "when creating a client connection",
+     ucs_offsetof(uct_rdmacm_cm_config_t, src_addr), UCS_CONFIG_TYPE_STRING},
+
+    {"TIMEOUT", "10s",
+     "Timeout for RDMA address and route resolve operations",
+     ucs_offsetof(uct_rdmacm_cm_config_t, timeout), UCS_CONFIG_TYPE_TIME},
+
+    {NULL}
+};
+
+static ucs_status_t
+uct_rdmacm_query_md_resources(uct_component_t *component,
+                              uct_md_resource_desc_t **resources_p,
+                              unsigned *num_resources_p)
+{
+    *resources_p     = NULL;
+    *num_resources_p = 0;
+    return UCS_OK;
+}
+
+uct_component_t uct_rdmacm_component = {
+    .query_md_resources = uct_rdmacm_query_md_resources,
+    .md_open            = ucs_empty_function_return_unsupported,
+#if HAVE_RDMACM_QP_LESS
+    .cm_open            = UCS_CLASS_NEW_FUNC_NAME(uct_rdmacm_cm_t),
+#else
+    .cm_open            = ucs_empty_function_return_unsupported,
```

Comment:
no HAVE_RDMACM_QP_LESS -> no component
and can remove all places we check for "unsupported"